### ISSUE TITLE
selendroid-test-app build failure

### ISSUE THREAD

**Issue by anandsadu** [appium/appium-uiautomator2-driver#4]
```
UNEXPECTED TOP-LEVEL EXCEPTION:
java.lang.RuntimeException: Exception parsing classes
 at com.android.dx.command.dexer.Main.processClass(Main.java:752)
 at com.android.dx.command.dexer.Main.processFileBytes(Main.java:718)
 at com.android.dx.command.dexer.Main.access$1200(Main.java:85)
 at com.android.dx.command.dexer.Main$FileBytesConsumer.processFileBytes(Main.java:1645)
 at com.android.dx.cf.direct.ClassPathOpener.processOne(ClassPathOpener.java:170)
 ... 25 more
Caused by: java.lang.IllegalArgumentException: bad utf-8 byte a0 at offset 00000004
 at com.android.dx.rop.cst.CstString.throwBadUtf8(CstString.java:171)
 at com.android.dx.rop.cst.CstString.utf8BytesToString(CstString.java:143)
 at com.android.dx.rop.cst.CstString.<init>(CstString.java:200)
 at com.android.dx.cf.cst.ConstantPoolParser.parseUtf8(ConstantPoolParser.java:371)
 ... 38 more
warning: Ignoring InnerClasses attribute for an anonymous inner class
(org.apache.xerces.impl.XMLEntityManager$1) that doesn't come with an
associated EnclosingMethod attribute. This class was probably produced by a
compiler that did not target the modern .class file format. The recommended
solution is to recompile the class from source, using an up-to-date compiler
and without specifying any "-target" type options. The consequence of ignoring
this warning is that reflective operations on this class will incorrectly
indicate that it is *not* an inner class.
warning: Ignoring InnerClasses attribute for an anonymous inner..
```

**Comment by bootstraponline** [comment]
I'm getting a different build error on master:


>$ cd selendroid-test-app
>$ gradle clean assembleDebug
>
>Execution failed for task ':selendroid-test->app:transformClassesAndResourcesWithExtractJarsForDebug'.
> com.android.build.transform.api.TransformException: java.lang.RuntimeException: java.io.FileNotFoundException: /appium-uiautomator2-driver/selendroid-test-app/build/intermediates/transforms/CLASSES_and_RESOURCES/EXTERNAL_LIBRARIES/extractJars/debug/selendroid-standalone-0.17.0-SNAPSHOT-with-dependencies.jar--727777440/META-INF/LICENSE (Is a directory)

**Comment by anandsadu** [comment]
Matthew , the issue that I had posted is happening on linux. I tried to build selendroid-test-app in windows and I could face the issue which you are facing. But on linux machine we are one step ahead.

:selendroid-test-app:compileDebugNdk UP-TO-DATE
:selendroid-test-app:compileDebugSources
:selendroid-test-app:transformClassesAndResourcesWithExtractJarsForDebug
:selendroid-test-app:transformClassesWithDexForDebug

on windows build is failing at transformClassesAndResourcesWithExtractJarsForDebug.
